xayn-ai/src/ltr/list_net/mod.rs

```diff
@@ -121,6 +122,52 @@ impl ListNet {
         }
     }
 
+    /// Create a new ListNet instance with random weights.
+    ///
+    /// The weights are initialized using the He-Normal weight
+    /// initializer, the biases are initialized to 0.
+    //TODO[maybe] run some tests to see weather or not Glorot or He
+    //            Uniform/Normal initialization works better for us.
```

Comment:
afair, Glorot works better for layers with sigmoid activations and He for layers with relu activations. regarding the distribution, i once read a benchmark which showed slight benefits of uniform over normal, but that might be anecdotal and model dependent, in the end it shouldn't matter much.

Reply:
I did a bit of ad-hoc research when deciding what to use and also came to the same result. But also found enough sources which:

- Show that the idea when Glorot or He is better don't always work out.
- The difference of Uniform and Normal is very little and supposedly papers which show one is better then the other are often not reproducible if just slightly different networks are used.

Either way TensorFlows default is Glorot Uniform and the soundgarden implementation does not change this, but when I train on the 500_000 user samples I always get a slightly better evaluation cost. E.g. 0.312... instead of 0.319... .

Now that I can use the GPU systems I could try running all for combinations (not in this PR, but based on my current working branch) it would be pretty simple.